### PR TITLE
EOS-26789: Add a small IO-test piece at the end of k8s Jenkins deployment job

### PR DESCRIPTION
**Pull request by nitisdev**
# Problem Statement
- Add a small IO-test piece at the end of k8s Jenkins deployment job

# Design
-  Added a small IO-test piece at the end of k8s Jenkins deployment job.

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- Single node: http://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/nitisdev/job/test_cortx_setup/157/console
- 3 node: http://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/nitisdev/job/test_cortx_setup/158/console

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [x] Interface change (if any) are documented
- [x] Side effects on other features (deployment/upgrade)
- [x] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: solutions/kubernetes/functions.sh [hunk 1]
```diff
@@ -18,6 +18,11 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
+function add_separator {
+    set +x +e
```
Comment (gauravchaudhari02) [inline solutions/kubernetes/functions.sh:22]: can't we combine both flags as `set +xe`?

File: solutions/kubernetes/cortx-deploy.sh [hunk 2]
```diff
@@ -111,6 +111,11 @@ EOF
         echo "---------------------------------------[ Print Cluster Status ]----------------------------------------------"
         rm -rf /var/tmp/cortx-cluster-status.txt
         ssh -o 'StrictHostKeyChecking=no' "$master_node" '/var/tmp/cortx-deploy-functions.sh --status' | tee /var/tmp/cortx-cluster-status.txt
+
+        # IO test
+        echo -e "\n\n\n----------------------[ Setting up IO testing ]-------------------------\n"
```
Comment (gauravchaudhari02) [inline solutions/kubernetes/cortx-deploy.sh:116]: use function `add_separator` here as well.

File: solutions/kubernetes/functions.sh (hunk 1 above)
Comment (nitisdev) [reply solutions/kubernetes/functions.sh]: Done

File: solutions/kubernetes/cortx-deploy.sh (hunk 2 above)
Comment (nitisdev) [reply solutions/kubernetes/cortx-deploy.sh]: Done

**Review by shailesh-vaidya — APPROVED** [review]

**Pull request merged**